src/pages/Member/AdminUpdateClass/DeleteClassModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { deleteClass } from '../../../api/apiPlanClass';
+
+import { Modal } from '../../../components/Modal/Modal';
+import { Button, PrimaryButton } from '../../../components/Button';
+import { ErrorMessageInterface } from '../../../types';
+import { StyledErrorMessage } from '../../../components/ErrorMessage';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
+
+const Text = styled.h4`
+  display: flex;
+  gap: 1em;
+  align-items: center;
+
+  span {
+    font-size: 1.5rem;
```

Comment:
I think this is too big, maybe instead, just have the font-weight as bold, but keep the font size the same as the other words in the sentence